_data/rohan2420/rohan2420.md

```diff
@@ -0,0 +1,7 @@
+---
+name: ROHAN PATNAIK # No longer than 28 characters
+institution: CET, BHUBANESWAR 🚩 # no longer than 58 characters
+profile_pic: Rohan.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.

```

Comment:
Hey @rohan2402 , the filename for your profile picture isn't the same as mentioned here. Please correct the typo